target/linux/generic/config-4.19

```diff
@@ -3681,6 +3681,7 @@ CONFIG_PARTITION_ADVANCED=y
 # CONFIG_PCIE_KIRIN is not set
 # CONFIG_PCIE_PTM is not set
 # CONFIG_PCIE_XILINX is not set
+# CONFIG_PCI_V3_SEMI is not set
```

Comment:
`kernel: generic: add missing configuration option`

Adding this config option, because foo...